include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -132,7 +151,37 @@ class CMAES
   //! Modify the selection policy.
   SelectionPolicyType& SelectionPolicy() { return selectionPolicy; }
 
+  //! Get the weight policy.
+  const WeightPolicyType& WeightPolicy() const { return weightPolicy; }
+  //! Modify the weight policy.
+  WeightPolicyType& WeightPolicy() { return weightPolicy; }
+
+  //! Get the update policy.
+  const UpdatePolicyType& UpdatePolicy() const { return updatePolicy; }
+  //! Modify the update policy
+  UpdatePolicyType& UpdatePolicy() { return updatePolicy; }
+
  private:
+  //! Initializing the parameters function
+  template<typename MatType>
+  void initialize(MatType& iterate);
+
+  //! Update the algorithm's parameters
+  template<typename MatType, typename BaseMatType>
+  void update(MatType& iterate,
```

Comment:
```suggestion
  void Update(MatType& iterate,
```

Please use capital casing for all names, see [conventions ](https://github.com/mlpack/mlpack/wiki/DesignGuidelines#naming-conventions) for more details.